scripts/pychop/arcs.yaml

```
@@ -106,15 +106,12 @@ moderator:
   name: Ambient Water           # A==water, AP==poisoned water, CH4==methane, H2==hydrogen. This is only used for analytical calculations
                                 # of the flux distribution for ISIS TS1 moderators. If measured_flux is defined below, name can be anything
   imod: 1                       # Moderator time profile type: 0==chi^2, 1==Old_Ikeda-Carpenter, 2==modified chi^2, 3==polynomial, 4==Tobyfit Ikeda-Carpenter
-  mod_pars: [119.63, 33.618,    # Parameters for time profile (for I-K is: [S1, S2, B1, B2, Emod]
-             .037, .17, 172.42] #    where tau_f=1/(sqrt(Ei)*E2V*sqrt(S1^2+(S2*lam)^2), tau_s=B1 for E<=130meV, B2 otherwise, R=exp(-Ei/Emod)
+  mod_pars: [281., 79.0,        # Parameters for time profile (for I-K is: [S1, S2, B1, B2, Emod]
+             .087, .4, 172.]    #    where tau_f=1/(sqrt(Ei)*E2V*sqrt(S1^2+(S2*lam)^2), tau_s=1/B1 for E<=130meV, 1/B2 otherwise, R=exp(-Ei/Emod)
+                                #    and the overall variance (not FWHM) is: tau^2 = 3/tau_f^2 + R/(2-R)/tau_s^2
```

```suggestion
                                #    and the overall variance (not FWHM) is: tau^2 = 3 tau_f^2 + R(2-R) tau_s^2
```

I think this is more accurate? May need updating in other files if so.